build-system/tasks/presubmit-checks.js

```diff
@@ -784,6 +784,9 @@ var forbiddenTermsSrcInclusive = {
       'dist.3p/current/integration.js',
     ],
   },
+  '\\.remove\\(\\)': {
```

Comment:
AFAICT only `remove` that does not take arguments is `Element.remove()` so we should be good here (at least for now). 

Having the rule in closure compiler is possible too but will miss many untyped cases.